packages/datadog-plugin-google-cloud-pubsub/test/index.spec.js

```diff
@@ -46,7 +46,7 @@ describe('Plugin', () => {
               meta: {
                 'pubsub.method': 'createTopic',
                 'span.kind': 'client',
-                'component': 'google-cloud-pubsub'
+                'component': '@google-cloud/pubsub'
```

Comment:
I thought the idea was to have the name of the component and not necessarily the exact package name in `npm`, which was reflected before this change?

Reply:
Fixed this. Within the google plugin, the start span call was explicitly setting the component tag to `@google-cloud/pubsub`. Now works as expected. Also fixed tests.